File: u3u2/Assets/Scripts/app/net/c2s/CGTestLong.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 测试
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGTestLong :BaseMessage
{
	
	/** 测试Long型 */
	private long testLong;
	/** 测试String型 */
	private string testString;
	
	public CGTestLong ()
	{
	}
	
	public CGTestLong (
			long testLong,
			string testString )
	{
			this.testLong = testLong;
			this.testString = testString;
	}
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{
	// 测试Long型
	WriteLong(testLong);
	// 测试String型
	WriteString(testString);

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_TEST_LONG;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}

File: server/当前版本/server/zlj1/game_server/src/com/imop/lj/gameserver/test/msg/CGTestLong.java
package com.imop.lj.gameserver.test.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.test.handler.TestHandlerFactory;

/**
 * 测试
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGTestLong extends CGMessage{
	
	/** 测试Long型 */
	private long testLong;
	/** 测试String型 */
	private String testString;
	
	public CGTestLong (){
	}
	
	public CGTestLong (
			long testLong,
			String testString ){
			this.testLong = testLong;
			this.testString = testString;
	}
	
	@Override
	protected boolean readImpl() {

	// 测试Long型
	long _testLong = readLong();
	//end


	// 测试String型
	String _testString = readString();
	//end



			this.testLong = _testLong;
			this.testString = _testString;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// 测试Long型
	writeLong(testLong);


	// 测试String型
	writeString(testString);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_TEST_LONG;
	}
	
	@Override
	public String getTypeName() {
		return "CG_TEST_LONG";
	}

	public long getTestLong(){
		return testLong;
	}
		
	public void setTestLong(long testLong){
		this.testLong = testLong;
	}

	public String getTestString(){
		return testString;
	}
		
	public void setTestString(String testString){
		this.testString = testString;
	}


	@Override
	public void execute() {
		TestHandlerFactory.getHandler().handleTestLong(this.getSession().getPlayer(), this);
	}
}